apps/condo/domains/property/schema/Property.js

```diff
@@ -249,6 +250,25 @@ const Property = new GQLListSchema('Property', {
             },
         },
 
+        ticketsDeferred: {
+            schemaDoc: 'Counter for deferred tickets',
+            type: Virtual,
+            resolver: async (item, _, context) => {
+                const { data, errors } = await context.executeGraphQL({
+                    query: GET_TICKET_DEFERRED_COUNT_BY_PROPERTY_ID_QUERY,
```

Comment:
why not use `count` from `generateServerUtils`?

Reply:
Done this